src/components/Link.js

```diff
@@ -11,7 +11,7 @@ const data = [{ url: 'https://google.com', linktext: 'Weiterlesen auf maz-online
 
 const LinkStyle = styled.Text`
   color: ${colors.secondary};
-  font-weight: bold;
+  font-family: titillium-web-bold;
```

Comment:
please add `font-weight: 600;` here, or is it bold anyway?

Reply:
it's already bold thanks to titillium-web-bold